src/metabase/public_settings.clj

```diff
@@ -462,33 +434,79 @@
   :visibility :public
   :type       :string
   :audit      :getter
-  :enabled?   premium-features/enable-whitelabeling?
+  :feature    :whitelabeling
   :default    "app/assets/img/logo.svg")
 
 (defsetting application-favicon-url
   (deferred-tru "The url or image that you want to use as the favicon.")
   :visibility :public
   :type       :string
   :audit      :getter
-  :enabled?   premium-features/enable-whitelabeling?
+  :feature    :whitelabeling
   :default    "app/assets/img/favicon.ico")
 
 (defsetting show-metabot
   (deferred-tru "Enables Metabot character on the home page")
   :visibility :public
   :type       :boolean
   :audit      :getter
-  :enabled?   premium-features/enable-whitelabeling?
+  :feature    :whitelabeling
   :default    true)
 
 (defsetting show-lighthouse-illustration
   (deferred-tru "Display the lighthouse illustration on the home and login pages.")
   :visibility :public
   :type       :boolean
   :audit      :getter
-  :enabled?   premium-features/enable-whitelabeling?
+  :feature    :whitelabeling
   :default    true)
 
+(def ^:private help-link-options
+  #{:metabase :hidden :custom})
+
+(defsetting help-link
+  (deferred-tru
+   (str
+    "Keyword setting to control whitelabeling of the help link. Valid values are `:metabase`, `:hidden`, and "
+    "`:custom`. If `:custom` is set, the help link will use the URL specified in the `help-link-custom-destination`, "
+    "or be hidden if it is not set."))
+  :default    :default
+  :type       :keyword
+  :audit      :getter
+  :visibility :public
+  :feature    :whitelabeling
+  :default    :metabase
+  :setter     (fn [value]
+                (when-not (help-link-options value)
+                  (throw (ex-info (tru "Invalid help link option")
```

Comment:
I tried to revert `:feature :whitelabeling` lines to test the other stuff and I got this error when trying to change the `help-link` setting from the ui:
```
[backend] 2023-12-06 13:02:09,658 ERROR middleware.log :: PUT /api/setting/help-link 500 6.2 ms (0 DB calls) 
[backend] {:via
[backend]  [{:type clojure.lang.ExceptionInfo,
[backend]    :message "Invalid help link option",
[backend]    :data {:value "custom", :valid-options #{:custom :hidden :metabase}},
[backend]    :at [metabase.public_settings$eval63935$fn__63938 invoke "public_settings.clj" 481]}],
```